### PR TITLE
Fix #22

### LINKED ISSUE #22: Fix paper-collapse-group can not observe "opend" changes in its paper-collapse-item children
Considering the following example 

```html
<paper-collapse-group>
	<paper-collapse-item id="item0" icon="icons:favorite" header="Item 1">
		Only on item will be opened inside groups
	</paper-collapse-item>
	<paper-collapse-item id="item1" icon="icons:info" header="Item 2" opened>
		Only on item will be opened inside groups
	</paper-collapse-item>
	<paper-collapse-item id="item2" icon="icons:help" header="<b>Header <u>with</u> HTML</b>">
		Only on item will be opened inside groups
	</paper-collapse-item>
</paper-collapse-group>
```

When any  `paper-collapse-item` is clicked, the `toggle` event is fired and `paper-collapse-group` can catch this event to close other items, but if we change the `opened` property for one of the items directly like `js this.$.item1.opened = true`
 `paper-collapse-group` will not be able to handle this cause the `toggle` event is not fired.

### PR DESCRIPTION
**Pull request by hyyan**

### CODE REVIEW

File: paper-collapse-item.html [hunk 1]
```diff
@@ -133,15 +134,18 @@
...
 		// Private methods
 		/**
 		 * Fired whenever the status is changed (opened/closed)
 		 *
 		 * @event toggle
 		 */
 		_toggleOpened: function(e) {
-			this.opened = !this.opened;
-			this.fire('toggle', this);
+			this.set('opened',!this.opened);
```
Comment (ronnyroeller) [inline paper-collapse-item.html]: Could please add a space after the comma (for code consistency)? Thanks

File: paper-collapse-group.html [hunk 2]
```diff
@@ -89,7 +89,7 @@
 			this._items.forEach(function(item) {
 				// Force all other items (aside the one triggering the event) to
 				// get closed
-				if (item !== e.target) {
+				if (item !== e.target && true === e.target.opened) {
```
Comment (ronnyroeller) [inline paper-collapse-group.html]: Could you please flip the operands => `e.target.opened === true` (for code consistency)?

File: paper-collapse-item.html (hunk 1 above)
Comment (hyyan) [reply paper-collapse-item.html]: Sure , I will do it later this day :)

File: paper-collapse-group.html (hunk 2 above)
Comment (hyyan) [reply paper-collapse-group.html]: Sure , I will do it later this day :)

**Comment by hyyan** [comment]
@ronnyroeller This is fixed

File: paper-collapse-item.html [hunk 3]
```diff
@@ -124,7 +124,8 @@
 			opened: {
 				type: Boolean,
 				reflectToAttribute: true,
-				notify: true
+				notify: true,
+				observer: '_opendChanged'
```
Comment (ankon) [inline paper-collapse-item.html:128]: Typo: 'openEd'.

**Pull request merged**